src/containers/Drawer/index.jsx

```diff
@@ -57,13 +57,31 @@ const useStyles = makeStyles((theme) => ({
     paddingTop: "1.5rem",
     minHeight: "100vh",
   },
+
+  brand_name: {
+    fontWeight: "100",
+    padding: "8px 0",
+    marginLeft: "5px",
+    borderBottom: "3px solid #2089dc",
```

Comment:
Pls remove this & replace with a <Divider>

Reply:
Ok removing it.